docs/content.zh/docs/dev/table/sql/update.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "UPDATE 语句"
+weight: 17
+type: docs
+aliases:
+- /zh/dev/table/sql/update.html
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# UPDATE 语句
+
+`UPDATE` 语句可以用于根据条件更新表的数据。
+
+<span class="label label-danger">注意</span> 目前, `UPDATE` 语句仅支持批模式, 并且要求目标表实现了 {{< gh_link file="flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/sink/abilities/SupportsRowLevelUpdate.java" name="SupportsRowLevelUpdate" >}} 接口。
+如果在一个没有实现该接口的表上执行 `UPDATE`，则会抛异常。目前Flink内置的插件还没有实现该接口。
```

Comment:
The comments in [`delete.md`](https://github.com/apache/flink/pull/22398/files#diff-39e9f52b0ea2a05f552c8a7ed95dd78a0e858d560cc31bd84fa4b1eb81880960) are also suitable for update